dor/providers/models.py

```diff
@@ -74,8 +74,8 @@ class StructMap:
 class PackageResource:
     id: uuid.UUID
     type: str
-    alternate_identifier: AlternateIdentifier
-    events: list[PreservationEvent]
+    alternate_identifier: list[AlternateIdentifier] = field(default_factory=list)
```

Comment:
I'd change it to `alternate_identifiers`

Reply:
Done, and behold the changelog ;-)